framework/doc/content/source/actions/Action.md

```diff
@@ -25,3 +113,13 @@ overridden. Both
 the `ContactAction` in the contact module, and `PorousFlowActionBase` in the
 porous flow module provide examples of overriding this method. For the reasons
 behind why this must be done in the action system, please see [RelationshipManager.md#rm_action].
+
```

Comment:
We recently add a capability of having Action parameters controllable. You might want to refer to that somewhere in this file.

Reply:
Is there existing doc on this?

Reply:
Yes, added in `Controllable Parameters Added by Actions` section of `framework/doc/content/syntax/Controls/index.md`.